packages/edit-site/src/store/actions.js

```diff
@@ -133,9 +128,13 @@ export const addTemplate =
  *
  * @param {Object} template The template object.
  */
-export const removeTemplate = ( template ) => {
-	return removeTemplates( [ template ] );
-};
+export const removeTemplate =
+	( template ) =>
+	async ( { registry } ) => {
+		await unlock( registry.dispatch( editorStore ) ).removeTemplates( [
```

Comment:
(minor question) Is the `async/await` structure necessary here? We're just delegating to a private action, so I'd've expected something like `removeTemplate = tpl => unlock( ... ).removeTemplates( [ tpl ] )`. Or is it to signal something to the reader?

Reply:
Yes, we can just return and I updated the code. But it is a fact that async await provides a better signal to the reader that the action is async, but we don't do it in other parts of the code so we can just return here as well.